fhir-server/src/main/java/com/ibm/fhir/server/resources/Capabilities.java

```diff
@@ -383,10 +408,55 @@ private CapabilityStatement buildCapabilityStatement() throws Exception {
         return conformance;
     }
 
+    private List<Rest.Resource.Interaction> buildInteractions(List<String> interactionConfig) throws FHIRPersistenceException {
```

Comment:
This is such a complicated stack to get here, it may be good to put a comment about the input and output as an ImplNote (not required, just a thought on how complex this Resource can be to build. )

Reply:
added some javadoc